ctapipe/io/eventsourcefactory.py

```diff
@@ -4,6 +4,7 @@
 # EventFileReader imports so that EventFileReaderFactory can see them
 # (they need to exist in the global namespace)
 import ctapipe.io.hessioeventsource
+from .import zfits
```

Comment:
Could you fix this import?

Reply:
Sorry, what I meant was could you rename zfits.py to zfitseventsource.py, and use an absolute reference to the module `import ctapipe.io.zfitseventsource`?

Reply:
rename file: very good.. I will ... better even to sst1meventsource.py, since I renamed the class to not imply any generality which might not exist.

Reply:
absolute import .. why?